primitives/account/src/logs.rs

```diff
@@ -0,0 +1,364 @@
+                    || new_reward_address
+                        .as_ref()
+                        .map(|new_reward_address| new_reward_address == address)
+                        .unwrap_or(false)
+            }
+            Log::InactivateValidator { validator_address } => validator_address == address,
+            Log::ReactivateValidator { validator_address } => validator_address == address,
+            Log::UnparkValidator { validator_address } => validator_address == address,
+            Log::CreateStaker {
+                staker_address,
+                validator_address,
+                ..
+            } => {
+                staker_address == address
+                    || validator_address
+                        .as_ref()
+                        .map(|validator_address| validator_address == address)
+                        .unwrap_or(false)
+            }
+            Log::Stake {
+                staker_address,
+                validator_address,
+                ..
+            } => {
+                staker_address == address
+                    || validator_address
+                        .as_ref()
+                        .map(|validator_address| validator_address == address)
+                        .unwrap_or(false)
+            }
+            Log::UpdateStaker {
+                staker_address,
+                old_validator_address,
+                new_validator_address,
+            } => {
+                staker_address == address
+                    || old_validator_address
+                        .as_ref()
+                        .map(|old_address| old_address == address)
+                        .unwrap_or(false)
+                    || new_validator_address
+                        .as_ref()
+                        .map(|new_address| new_address == address)
+                        .unwrap_or(false)
+            }
+            Log::DeleteValidator {
+                validator_address,
+                reward_address,
+            } => validator_address == address || reward_address == address,
+            Log::Unstake {
+                staker_address,
+                validator_address,
+                ..
+            } => {
+                staker_address == address
+                    || validator_address
+                        .as_ref()
+                        .map(|validator_address| validator_address == address)
+                        .unwrap_or(false)
+            }
+            Log::PayoutReward { to, .. } => to == address,
+            Log::Park {
+                validator_address, ..
+            } => validator_address == address,
+            Log::Slash {
+                validator_address, ..
+            } => validator_address == address,
+            Log::RevertContract { contract_address } => contract_address == address,
+        }
+    }
+}
+
+#[derive(Debug, PartialEq, Eq, Clone)]
+#[cfg_attr(feature = "serde-derive", derive(serde::Serialize, serde::Deserialize))]
```

Comment:
Aren't we missing `#[serde(rename_all = "camelCase")]`?

Reply:
It's unnecessary since I chose to rename the only field that would be affected with its own rename rule already.

Reply:
I see your point but I'd still added in case we add more members in the future and we forget to revise it